src/DataLoaders.py

```diff
@@ -0,0 +1,102 @@
+# Classes for wrapping data into datasets
+
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from tqdm import tqdm
+from typing import Optional, Literal
+from torch_geometric.data import Dataset
+
+class ProteinDataset(Dataset):
+    """
+    Dataset for loading proteins from the paper given splits.
+    """
+    def __init__(self, labels, database, include, flex: Literal["msqf", "bfact", "pseudo"] = "msqf", exclude: Optional[str] = None):
+        self.flex = flex
+        self.database = database
+        self.structures = []
+        self.protein_labels = pd.DataFrame(columns = ['chain_id', 'label'])
+        if exclude != None:
+            structure_set = set(line.strip() for line in open(include)).difference(set(line.strip().replace("_",".") for line in open(exclude)))
+        else:
+            structure_set = set(line.strip() for line in open(include))
+        for id in tqdm(structure_set):
+            PDBid, chain = id.split(".")
+            if(PDBid in database):
+                structure = database[PDBid]["structure"]
+                protein_chain = structure[(structure.chain_id == chain) & struc.filter_amino_acids(structure)] 
+                if(check_chain(protein_chain)):
```

Comment:
For consistent code formating, I highly recommend you use an autoformatter such as [black](https://black.readthedocs.io/en/stable/the_black_code_style/index.html) -- this is what is used in professional python code.